src/utils/fs-helpers.js

```diff
@@ -0,0 +1,45 @@
+import * as path from 'path';
+import { createTempDir } from 'broccoli-test-helper';
```

Comment:
do we need this?
creating a tmp directory is pretty easy without a library: https://github.com/NullVoxPopuli/ember-apply/blob/main/ember-apply/src/test-utils.js#L9

Reply:
createTempDir also provide way to destroy lib (automatically), but I agree, we could simplify path here

Reply:
we can just leave things in tmp forever, yeah?
Operating system will clean up for us

Reply:
@NullVoxPopuli I forgot :) result of createTempDir has `write` function, to write JSON-serialized FS structure to file-system, it will be tricky to replace

Reply:
https://github.com/broccolijs/broccoli-test-helper/blob/master/docs/api/index.md#tempdir